hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -794,16 +794,16 @@ def get_bank_member_signals_to_process_page(
         self,
         signal_type: t.Type[SignalType],
         exclusive_start_key: t.Optional[DynamoDBCursorKey] = None,
-        limit: int = 100,
+        limit: t.Optional[int] = None,
```

Comment:
Instead of imposing a client side arbitrary limit, we now let DynamoDB decide how big a page it can send in one query.

This allows for far better performance. With a size of 100, the round-trip latencies added up and made it impossible to build an index for a million entries in a single indexing lambda run. With removing the limit, we've reached peak performance. :)

Reply:
Thanks for the explainer! Will we ever get into a position where the index cannot be built in a single lambda run?